examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -214,5 +214,6 @@
   ],
   "labels": {
     "template": "mongodb-ephemeral-template"
-  }
+  },
+  "message": "You can connect to the database using MongoDB connection URL mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@${DATABASE_SERVICE_NAME}/${MONGODB_DATABASE}"
```

Comment:
I think it does always work? For example, in Java:

http://api.mongodb.com/java/2.10.1/com/mongodb/MongoClientURI.html